charts/vald/values/vald-backup-via-pv-and-s3.yaml

```
@@ -0,0 +1,98 @@
+defaults:
+  image:
+    tag: latest
+
+gateway:
+  lb:
+    resources:
+      requests:
+        cpu: 100m
+        memory: 50Mi
+    gateway_config:
+      # number of index replica.
+      index_replica: 2
+
+agent:
+  minReplicas: 6
+  maxReplicas: 6
+  podManagementPolicy: Parallel
+  resources:
+    requests:
+      cpu: 100m
+      memory: 50Mi
+    limits:
+      memory: 1Gi
+  # We recommend to set this value long enough to ensure the backup speed of S3 and PV, since the Index is backed up at the end of the pod.
+  terminationGracePeriodSeconds: 3600
+  # This is the persistent volume settings.
+  # Please change it according to your environment.
+  persistentVolume:
+    enabled: true
+    accessMode: ReadWriteOnce
+    storageClass: local-path
+    size: 2Gi
+  ngt:
+    dimension: 784
+    index_path: "/var/ngt/index"
+    enable_in_memory_mode: false
+    # limit duration of automatic indexing.
+    auto_index_duration_limit: 24h
+    # check duration of automatic indexing.
+    auto_index_check_duration: 5h
+    # number of cache to trigger automatic indexing.
+    auto_index_length: 100
+    # duration of automatic save index.
+    auto_save_index_duration: 10h
+    # batch process pool size of automatic create index operation.
+    auto_create_index_pool_size: 1000
```

this should be more long duration like below because index-manager can handle it.
```suggestion
    auto_index_duration_limit: 730h
    # check duration of automatic indexing.
    auto_index_check_duration: 24h
    # number of cache to trigger automatic indexing.
    auto_index_length: 1000
    # duration of automatic save index.
    auto_save_index_duration: 365h
    # batch process pool size of automatic create index operation.
    auto_create_index_pool_size: 1000
```